### PR TITLE
Add Alpha (Balance Parameter) To ALC Interface

### LINKED ISSUE #29798: Add alpha to ALC
Want to add a way for users to choose an alpha value in the ALC

### PR DESCRIPTION
**Pull request by Matt-Cumber**
~~**Merge #29847  first**~~ **Merged**

**Description of work.**
Added new option to ALC interface to set alpha value

**To test:**
1. Open ALC
2. Load some single period data and check alpha is available (MUSR 62260)
3. Load some multi period data and check alpha is not available (MUSR 15189)

Fixes #29798 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Muon ALC.rst [hunk 1]
```diff
@@ -192,7 +193,8 @@ Dead Time Correction
 Grouping
   Detector grouping to apply. *Auto* will load the grouping information contained 
   in the run file, while *Custom* allows to specify the list of spectra for both the 
-  forward and backward groups.
+  forward and backward groups. *Alpha* is the balanace parameter and is only available
```
Comment (AnthonyLim23) [inline docs/source/interfaces/Muon ALC.rst:196]:
```suggestion
  forward and backward groups. **Alpha** is the balance parameter and is only available 
```

**Review by AnthonyLim23 — CHANGES_REQUESTED** [review]

**Comment by AnthonyLim23** [comment]
GUI works well

**Review by AnthonyLim23 — DISMISSED** [review]

**Review by DavidFair — DISMISSED** [review]
I've resolved the merge conflict in the docs for you, and I'm happy to :shipit: once the CI finishes (to make sure I haven't botched the docs somehow)

**Review by DavidFair — APPROVED** [review]
3rd time lucky on getting this merged before another merge conflict develops....

**Comment by AnthonyLim23** [comment]
@DavidFair could you please merge this before we get another conflict?

**Pull request merged**